src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Workloads/IWpfWorkloadDescriptorDataSource.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Workloads
+{
+    [ProjectSystemContract(ProjectSystemContractScope.ConfiguredProject, ProjectSystemContractProvider.Private, Cardinality = Composition.ImportCardinality.ExactlyOne)]
+    internal interface IWpfWorkloadDescriptorDataSource : IProjectValueDataSource<ISet<WorkloadDescriptor>>
```

Comment:
Please add a doc comment explaining the purpose of this type.

Reply:
done